fuel-core/Cargo.toml

```diff
@@ -43,6 +43,7 @@ fuel-core-interfaces = { path = "../fuel-core-interfaces", version = "0.9.5", fe
 fuel-relayer = { path = "../fuel-relayer", version = "0.9.5" }
 fuel-sync = { path = "../fuel-sync", version = "0.9.5" }
 fuel-txpool = { path = "../fuel-txpool", version = "0.9.5" }
+fuel-types = "0.5"
```

Comment:
This should be re-exported by `fuel-core-interfaces`. If we let these fuel-* deps proliferate into the `Cargo.toml` of all our crates, we increase the likelihood of a missed version bump. It also just makes a lot of extra work when it's time to upgrade these versions if we have to update in several places instead of just `fuel-core-interfaces`.